packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -172,12 +231,11 @@ export class InsuredBridgeL1Client {
   }
 
   async calculateRealizedLpFeePctForDeposit(deposit: Deposit): Promise<BN> {
-    if (this.rateModels === undefined || this.rateModels[deposit.l1Token] === undefined)
```

Comment:
An error will now be thrown in `getRateModelForBlockNumber` if a rate model isn't found for the token